app/content/errors/404.html

```diff
@@ -0,0 +1,19 @@
+<main id="content" role="main" class="group">
+  <header class="page-header group">
+    <div>
+      <h1>Page not found</h1>
+    </div>
+  </header>
+  <div class="article-container">
+    <article role="article" class="group">
+      <div class="inner">
+        <p>If you entered a web address please check it was correct.</p>
+        <p>You can also <a href="/">browse from the homepage</a> to find the information you need.</p>
+      </div>
+    </article>
+  </div>
+
+  <script type="text/javascript">
+  window.httpStatusCode = '404';
```

Comment:
Looks like has been taken from the GOV.UK page.  Think we don't need it in this case, and also there's a duplication of status codes, as it is being sent here on the client side but also on the server side in the code further down where res.status is used.